README.md

```diff
@@ -17,5 +17,7 @@ No installation instructions at the moment.
 No usage instructions at the moment.
 
 ## Development
-
-No development instructions at the moment.
\ No newline at end of file
+- There is no possibility to test Prow configuration locally, on minikube. All tests needs to be done on clusters. 
```

Comment:
~There is no possibility to~ `You cannot test Prow configuration locally on Minikube. Perform all the tests on the cluster.`